next/components/cards/FileRowCard.tsx

```diff
@@ -19,37 +21,50 @@ const FileRowCard = ({
   size,
   className,
   ariaLabel,
+  variant,
```

Comment:
I called the prop `hideBottomDivider` since we already use this naming in other components. For now, the component renders the `HorizontalDivider` component, but it can be done conditionally, as in `SearchResultCard`